### DISCUSSION TITLE
how to extract the simulation data

### DISCUSSION THREAD

**Discussion by nibiyoule** [pybamm-team/PyBaMM#1485] [Q&A] [answered]
Actually, i did some parameter initialization on the pybamm model, and I wanted to compare the experimental value with the simulation value. However, how to extract the "Terminal voltage" data or "lithium ion concentration "data from the model, because i want to do error analysis with my experimental value.

**Comment by valentinsulzer** [comment, marked as answer]
What do you mean by "extract"?

**Reply by nibiyoule** [reply]
Thank you for your reply!
I mean that i want to get the very terminal voltage data list or array of output variables in the simulation, not only a volatage figure. I don't know how to get the data of voltage figure (specify voltage), it didn't show in the variables area like in the Matlab, i want to get the data for caculation.
Hope for your response.

**Reply by valentinsulzer** [reply]
`solution["Terminal voltage [V]"].entries`
See also https://github.com/pybamm-team/PyBaMM/blob/develop/examples/notebooks/solution-data-and-processed-variables.ipynb

**Reply by nibiyoule** [reply]
Thank you very much for your valuable time. Your suggestions are really helpful. Maybe I should take more time to know more about this examples.
